include/ola/client/OlaClient.h

```diff
@@ -130,6 +131,16 @@ class OlaClient {
                            CandidatePortsCallback *callback);
 
   /**
+   * @brief Set the state of a plugin.
+   * @param plugin_ids the ids of the plugins to set the state of.
+   * @param state the state to set the plugin to
+   * @param callback the SetCallback to invoke upon completion.
+   */
+  void SetPluginState(const std::vector<ola_plugin_id> &plugin_ids,
```

Comment:
Make this take a single ID.

The problem with passing multiple plugins in a single call is how we handle failures. Do we change all or none? That means we have to handle rollbacks.  It gets complicated.